src/hacbs/components/Commits/__mocks__/pipeline-with-commits.ts

```diff
@@ -0,0 +1,725 @@
+import { PipelineRunKind } from '../../../types';
+
+export const pipelineWithCommits: PipelineRunKind[] = [
+  {
+    apiVersion: 'tekton.dev/v1beta1',
+    apiGroup: 'tekton.dev',
+    kind: 'PipelineRun',
+    metadata: {
+      annotations: {
+        'build.appstudio.openshift.io/image':
+          'quay.io/redhat-appstudio/user-workload:abhindas-java-springboot-sample',
+        'build.appstudio.openshift.io/repo':
+          'https://github.com/devfile-samples/devfile-sample-java-springboot-basic.git',
+        'chains.tekton.dev/retries': '3',
+        'chains.tekton.dev/signed': 'failed',
+        'chains.tekton.dev/transparency':
+          'http://rekor-server.enterprise-contract-service.svc:3000/api/v1/log/entries?logIndex=3143',
+        'results.tekton.dev/record':
+          'abhindas/results/558677f8-e01c-4d15-8729-5e838bd492aa/records/558677f8-e01c-4d15-8729-5e838bd492aa',
+        'results.tekton.dev/result': 'abhindas/results/558677f8-e01c-4d15-8729-5e838bd492aa',
+        'build.appstudio.redhat.com/commit_sha': 'commit123',
+        'build.appstudio.redhat.com/target_branch': 'branch_1',
+      },
+      creationTimestamp: '2022-07-19T11:35:46Z',
+      generation: 1,
+      labels: {
+        'build.appstudio.openshift.io/application': 'purple-mermaid-app',
+        'build.appstudio.openshift.io/build': 'true',
+        'build.appstudio.redhat.com/component': 'sample-component',
+        'build.appstudio.redhat.com/user': 'abhi',
+      },
+      name: 'java-springboot-sample-x778q',
+      namespace: 'abhindas',
```

Comment:
Change it to `test` or `test-ns` all the occurrences
```suggestion
      namespace: 'test-ns',
```

Reply:
updated